Optimize SpatialBNOp on GPU

xiaomengy opened the pull request:
Summary:
Optimize SpatialBNOp on GPU

i-am-not-moving-c2-to-c10

Differential Revision: D13829833

bddppq commented:
To fix the amd build errors, rename this line accordingly https://github.com/pytorch/pytorch/blob/f6e6b0fd33c7e9c33e15fecd37953aea4065302a/tools/amd_build/pyHIPIFY/cuda_to_hip_mappings.py#L2251

houseroad reviewed (approved):
LGTM

xiaomengy committed:
Optimize SpatialBNOp on GPU (#16395)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/16395

Optimize SpatialBNOp on GPU

i-am-not-moving-c2-to-c10

Reviewed By: houseroad

Differential Revision: D13829833

fbshipit-source-id: 734c04ae1850e82970324b39b05bb035b2ac4871
